cli/azd/pkg/tools/powershell/powershell.go

```diff
@@ -25,7 +25,15 @@ type powershellScript struct {
 // Executes the specified powershell script
 // When interactive is true will attach to stdin, stdout & stderr
 func (bs *powershellScript) Execute(ctx context.Context, path string, options tools.ExecOptions) (exec.RunResult, error) {
-	runArgs := exec.NewRunArgs("pwsh", path).
+
+	cmdConfig := " -NoProfile"
+	normalCmd := "pwsh"
+
+	if !options.IsRunProfile {
```

Comment:
Sort of wondering if instead of adding `IsRunProfile` and setting it in `hooks.go`, I wonder if we should be doing it here, so all invocations of `pwsh` get it out of the box?  I'm sort of wondering if we ever want to not do this for pwsh.

Reply:
@ellismg In the current version of azd, pwsh does not support the `-NoProfile` parameter because in the `exec.NewRunArgs("pwsh", path)` method in the [powershell.go](https://github.com/Azure/azure-dev/blob/main/cli/azd/pkg/tools/powershell/powershell.go#L28) file, the default is not have the -`NoProfile` parameter, and the hard-coded value is `pwsh`, as follow:
![image](https://github.com/user-attachments/assets/206de913-4acb-49dc-96a9-e31f4b22ba4f)
 
So if we want to support the `-NoProfile` parameter, we can only add `IsRunProfile` here.